modules/ocf_printhost/files/enforcer

```diff
@@ -25,14 +25,14 @@ from textwrap import dedent
 from traceback import format_exc
 
 import ocflib.printing.quota as quota
+import redis
```

Comment:
I have a feeling pre-commit won't like this unordered import

Reply:
Well, seems like the commits have been passing the CI tests?